portals/publisher/source/src/app/components/Apis/Details/Environments/Environments.jsx

```diff
@@ -1397,6 +1418,17 @@ export default function Environments() {
 
     return (
         <>
+            {api.advertiseInfo && api.advertiseInfo.advertised && (
+                <MuiAlert severity='info'>
```

Comment:
add clear: both style and margin top

Reply:
Done with https://github.com/wso2/apim-apps/pull/130/commits/672d2559070933aed4d790145c5135ccd63e9dfb